gitnamehere/point-of-sale-group-project
# POS Navigation and account authentication

This took me wayyy too long

This PR adds in navigational buttons in the POS, adds and modify tables, and adds account authentication

(styling can be fixed later in a different pr)

Testing:
* make a new copy of `env.json` from `sample-env.json` and fill in relevant informtation
* `npm i`
* `npm run setup-demo`
* `npm start`
* open a new terminal and log in to the database with psql
* \c into the pos database, and type in `SELECT * FROM accounts`;
* verify the account with the credentials in the env file is inserted (the password will be hashed)
* go to http://localhost:3000/pos/ordering/
* see that a redirect page is displayed, click on the redirect link
* log in based on the provided credentials in the env.json file, notably boss_user and boss_password
* verify that login works and you can access the pos as normal
* click on the logout button in the POS home screen on the top right (don't mind the styling)
* verify that logging out works, and it redirects you to the login screen
* log back in, and go to http://localhost:3000/pos/ordering/
* test out the navigational buttons in the navbar, and verify that it works 
* go to http://localhost:3000/pos/item-management/create.html
* test out the navigational buttons, and verify that it works normally
* go to http://localhost:3000/pos
* test out that some of the buttons navigates you correctly
* restart the node server
* verify that you are still authenticated

time to sleep
